CHANGELOG.md

```diff
@@ -1,16 +1,25 @@
 ## Unreleased
 
 Bugs:
-* Helm Chart: fix deployment templating so setting `controller.kubernetesClusterDomain` works as defined in values.yaml. [GH-183](https://github.com/hashicorp/vault-secrets-operator/pull/183)
+* Helm Chart: fix deployment templating so setting `controller.kubernetesClusterDomain` works as defined in values.yaml: [GH-183](https://github.com/hashicorp/vault-secrets-operator/pull/183)
+* VaultPKISecret (VPS): Ensure `Spec.AltNames`, and `Spec.IPSans`are properly formatted for the Vault request: [GH-130](https://github.com/hashicorp/vault-secrets-operator/pull/130)
+* VaultPKISecret (VPS): Make `Spec.OtherSANS` a string slice (**breaking change**): [GH-190](https://github.com/hashicorp/vault-secrets-operator/pull/190)
```

Comment:
I could also see this going into a separate `Changes` section, to make it more obvious when there's a breaking change.

Reply:
Added in https://github.com/hashicorp/vault-secrets-operator/pull/193/commits/c16a3dd06597cda45ce3f1fdf09dd6e3ab3c48f5